builder/vmware/common/driver_fusion6.go

```diff
@@ -124,6 +125,24 @@ func (d *Fusion6Driver) Verify() error {
 	return compareVersions(matches[1], VMWARE_FUSION_VERSION, "Fusion Professional")
 }
 
+func (d *Fusion6Driver) ToolsIsoPath(k string) string {
+	// Fusion 13.x.x changes tools iso location
+	vmxpath := filepath.Join(d.AppPath, "Contents", "Library", "vmware-vmx")
+	var stderr bytes.Buffer
+	cmd := exec.Command(vmxpath, "-v")
+	cmd.Stderr = &stderr
+	versionRe := regexp.MustCompile(`(?i)VMware [a-z0-9-]+ (\d+)\.`)
+	matches := versionRe.FindStringSubmatch(stderr.String())
+	if matches[1] < "13" {
```

Comment:
I executed on Fusion 13 and found the plugin to crash because of an out range index error

```
~>  packer build -only=vmware-iso.vm -var-file=os_pkrvars/windows/windows-2022-x86_64.pkrvars.hcl ./packer_templates/
vmware-iso.vm: output will be in this color.

Build 'vmware-iso.vm' errored after 63 milliseconds 442 microseconds: unexpected EOF

==> Wait completed after 63 milliseconds 495 microseconds

==> Some builds didn't complete successfully and had errors:
--> vmware-iso.vm: unexpected EOF

==> Builds finished but no artifacts were created.
panic: runtime error: index out of range [1] with length 0
2023/02/10 14:04:53 packer-plugin-vmware plugin:
2023/02/10 14:04:53 packer-plugin-vmware plugin: goroutine 14 [running]:
2023/02/10 14:04:53 packer-plugin-vmware plugin: github.com/hashicorp/packer-plugin-vmware/builder/vmware/common.(*Fusion6Driver).ToolsIsoPath(0x14000a040c0, {0x140004dce80, 0x7})
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/Users/dev/Development/packer-plugin-vmware/builder/vmware/common/driver_fusion6.go:136 +0x414
2023/02/10 14:04:53 packer-plugin-vmware plugin: github.com/hashicorp/packer-plugin-vmware/builder/vmware/common.(*StepPrepareTools).Run(0x14000a28000, {0x0?, 0x5000?}, {0x10222aa20, 0x140002b9ec0})
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/Users/dev/Development/packer-plugin-vmware/builder/vmware/common/step_prepare_tools.go:30 +0xcc
2023/02/10 14:04:53 packer-plugin-vmware plugin: github.com/hashicorp/packer-plugin-sdk/multistep.(*BasicRunner).Run(0x14000a28060, {0x10222a5c0, 0x14000a04080}, {0x10222aa20, 0x140002b9ec0})
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/Users/dev/Development/go/pkg/mod/github.com/hashicorp/packer-plugin-sdk@v0.3.2/multistep/basic_runner.go:70 +0x2d4
2023/02/10 14:04:53 packer-plugin-vmware plugin: github.com/hashicorp/packer-plugin-vmware/builder/vmware/iso.(*Builder).Run(0x140000e5000, {0x10222a5c0, 0x14000a04080}, {0x10222c130?, 0x140002b8d50}, {0x10221e0b8?, 0x14000658020})
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/Users/dev/Development/packer-plugin-vmware/builder/vmware/iso/builder.go:200 +0x1610
2023/02/10 14:04:53 packer-plugin-vmware plugin: github.com/hashicorp/packer-plugin-sdk/rpc.(*BuilderServer).Run(0x1400003c040, 0xf6f99c?, 0x14000780e8c)
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/Users/dev/Development/go/pkg/mod/github.com/hashicorp/packer-plugin-sdk@v0.3.2/rpc/builder.go:117 +0x1e8
2023/02/10 14:04:53 packer-plugin-vmware plugin: reflect.Value.call({0x1400009e960?, 0x140003060f0?, 0x13?}, {0x101aa9a49, 0x4}, {0x14000331ef8, 0x3, 0x3?})
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/usr/local/go/src/reflect/value.go:556 +0x5e4
2023/02/10 14:04:53 packer-plugin-vmware plugin: reflect.Value.Call({0x1400009e960?, 0x140003060f0?, 0x100014000969ec8?}, {0x14000969ef8, 0x3, 0x3})
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/usr/local/go/src/reflect/value.go:339 +0x98
2023/02/10 14:04:53 packer-plugin-vmware plugin: net/rpc.(*service).call(0x1400003c080, 0x8?, 0x0?, 0x14000780030, 0x140000aa200, 0x1400010e778?, {0x101ec9dc0?, 0x14000780e88?, 0x1400010e720?}, {0x101e8dde0?, ...}, ...)
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/usr/local/go/src/net/rpc/server.go:381 +0x218
2023/02/10 14:04:53 packer-plugin-vmware plugin: created by net/rpc.(*Server).ServeCodec
2023/02/10 14:04:53 packer-plugin-vmware plugin: 	/usr/local/go/src/net/rpc/server.go:478 +0x378
```